env.d.ts

```diff
@@ -912,13 +971,13 @@ declare const $db: {
 	/** Find documents in a collection or view and returns a cursor to the selected documents.
 	  * @param query Specifies deletion criteria using query operators.
 	  * @param projection Specifies the fields to return in the documents that match the query filter. */
-	f: (query?: Query, projection?: Projection) => Cursor
+	f: <T extends MongoSchema = MongoSchema>(query?: Query<T>, projection?: Projection<T>) => Cursor<MongoDocument<T>>
```

Comment:
Although convenient, it is not type safe to assume the record you get back from the database is going to be the type you think it is. The type returned from `$db.f()` is vague for the same reason that I set `args: unknown`. For type safe scripts, people should validate the types first.